checker/build.gradle

```diff
@@ -11,7 +11,7 @@ sourceSets {
     main {
         resources {
             // Stub files, message.properties, etc.
-            srcDirs += ['src/main/java']
+            srcDirs = ['src/main/java']
```

Comment:
This is an odd change that doesn't seem to have anything to do with the rest of this PR. What's the reasoning behind it? I don't think I can merge this in good conscience without knowing why this change is needed.